favourite-colour/README.md

```diff
@@ -1,15 +1,15 @@
 # Favourite Colour
 
-Shows and stores your favourite colour, in storage.sync in the about:addons page for the add-on.
+Shows and stores your favourite colour, in storage.sync in the extension's about:addons page.
 
 Demonstrates:
 
 * storing data with storage.sync
+* reading data from storage.managed
+* creating an options page and opening it with `runtime.openOptionsPage()`
+* best practice for supporting automatic theme switching in the options page
```

Comment:
```suggestion
* best practice for supporting automatic theme switching (dark theme) in the options page
```

This feature is more well known as "dark theme support", hence my suggestion to include that term here.